Eastern-Research-Group/mywaterway
# Feature/update survey map file

## Related Issues:
* https://jira.epa.gov/browse/HMW-58

## Main Changes:
* Updated the surveyMapping.json file
* Updated the stateNationalUses.json file

## Steps To Test:
1. Navigate to http://localhost:3000/state/CO/water-quality-overview
2. Click the `Aquatic Life` button
3. Select `Rivers and Streams` in the `Water Type` input
4. Verify the surveys section displays (i.e., pie chart visible)
5. Repeat similar steps for Massachusetts and Pennsylvania,
